readthedocs/api/base.py

```diff
@@ -223,7 +221,11 @@ def get_anchor(self, request, **kwargs):
         self.throttle_check(request)
 
         query = request.GET.get('q', '')
-        redis_data = redis_client.keys("*redirects:v4*%s*" % query)
+        try:
+            redis_client = cache.get_client(None)
```

Comment:
Seems similar to https://docs.djangoproject.com/en/1.8/topics/cache/#django.core.cache.get_cache -- which is deprecated.